src/coins.cpp

```diff
@@ -229,8 +250,20 @@ bool CCoinsViewCache::Flush() {
 void CCoinsViewCache::Uncache(const COutPoint& hash)
 {
     CCoinsMap::iterator it = cacheCoins.find(hash);
+
     if (it != cacheCoins.end() && it->second.flags == 0) {
         cachedCoinsUsage -= it->second.coin.DynamicMemoryUsage();
+
+        TRACE7(utxocache, uncache,
+            hash.hash.ToString().c_str(),
+            (uint32_t)hash.n,
+            (uint32_t)it->second.coin.nHeight,
+            (int64_t)it->second.coin.out.nValue,
+            (uint)it->second.coin.fCoinBase,
```

Comment:
```suggestion
            (bool)it->second.coin.fCoinBase,
```

Reply:
```suggestion
            (bool)it->second.coin.IsCoinBase(),
```

https://github.com/bitcoin/bitcoin/blob/b3a2b8c29fda5d924726d3087535bd34af7012b6/src/coins.h#L55-L57